script/tool/test/xctest_command_test.dart

```diff
@@ -314,14 +331,13 @@ void main() {
       });
 
       test('skip if macOS is implemented in a federated package', () async {
-        createFakePlugin('plugin', packagesDir, extraFiles: <String>[
-          'example/test',
-        ], platformSupport: <String, PlatformSupport>{
-          kPlatformMacos: PlatformSupport.federated,
-        });
+        createFakePlugin('plugin', packagesDir,
+            platformSupport: <String, PlatformSupport>{
+              kPlatformMacos: PlatformSupport.federated,
+            });
 
-        final List<String> output = await runCapturingPrint(runner,
-            <String>['xctest', '--macos', _kDestination, 'foo_destination']);
```

Comment:
Opportunistic cleanup; I accidentally left the targets when initially copy/pasting iOS tests for macOS.